message_app_client/src/components/usersTable/UsersTableDetails.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect, useContext } from "react";
+import TableRow from "./TableRow";
+import { SessionDataContext } from "../../Contexts";
+import { useNavigate } from "react-router-dom";
+import { getApiRoute, routes } from "../../state";
+
+const BASE_URL = getApiRoute("requestMessage")
+const UsersTableDetail = (props) => {
+    const { userId } = props;
+    const [ users, setUsers ] = useState([]);
+    const [ isFetchSuccess, setIsFetchSuccess ] = useState(true);
+    const { currentUserID, setCurrentRoute } = useContext(SessionDataContext);
+    const navigate = useNavigate();
+    const apiUrl = `${BASE_URL}/get_people/${currentUserID}`;
+
+    const fetchUsers = async () => {
+        const response = await fetch(apiUrl)
+        .catch(() => {
+            setIsFetchSuccess(false);
+        });
+        const usersData = await response.json();
+
+        setUsers(usersData);
+    };
+
+    useEffect(() => {
+        if (!currentUserID) {
+            setCurrentRoute("/find_users");
```

Comment:
Why do we need to set the current route here if we navigate to login route below?

Reply:
if the user goes to /find_users route without login, it will navigate to the login page. After login I want it to move back to /find_users. By default, after login, it will navigate the messages page. See the LoginPage.js below